DW-727: Add troubleshooting info of the confluence page (DOCS-12701) (#87)
mulesoft/docs-dataweave: modules/ROOT/pages/dataweave-memory-management.adoc

DataWeave keeps small files in memory, but after a configurable threshold uses disk space to avoid running out of memory. The created buffer files are placed in a default temporary directory. If you want to store those files in a custom directory instead, you can specify the directory by using the `java.io.tmpdir` property.

> Those buffer files remain in the temporary directory until the streams referencing them are closed. Note that the closure normally happens when flows complete their execution, so long-running and particularly concurrent executions could involve having many of these files living at once.

Reviewer: ```suggestion
The buffer files remain in the temporary directory until the streams that reference them are closed. Because closure normally happens when flows complete their execution, long-running and concurrent executions could involve having many of these files living at once.
```

Author: Added this with `so many buffer files in the temporary folder`
